Analysis/Tasks/PWGHF/qaTaskSimple.cxx

```diff
@@ -121,9 +119,13 @@ struct QAGlobalObservables {
 
   void process(const o2::aod::Collision& collision, const o2::aod::Tracks& tracks)
   {
-    if (collision.numContrib() < nMinNumberOfContributors.value) {
+    if (collision.numContrib() < MinNumberOfContributors.value) {
       return;
     }
+    int nTracks(0);
+    for (const auto& track : tracks) {
+      nTracks++;
+    }
```

Comment:
The tracks should be grouped by collision. Why don't you just take `tracks.size()`?